README.md

```diff
@@ -260,14 +307,55 @@ julia> f1(n) = @view A[:, 1];
 
 julia> f2(n) = @view A[:, 2];
 
-julia> x = readsas("productsales.sas7bdat", include_columns=[:ACTUAL,:PREDICT], 
-              number_array_fn=Dict(:ACTUAL => f1, :PREDICT => f2));
-Read productsales.sas7bdat with size 1440 x 2 in 0.32306 seconds
+julia> readsas("productsales.sas7bdat", include_columns=[:ACTUAL,:PREDICT], 
+               number_array_fn=Dict(:ACTUAL => f1, :PREDICT => f2));
+Read productsales.sas7bdat with size 1440 x 2 in 0.00041 seconds
+
+julia> A[1:5,:]
+5×2 Array{Float64,2}:
+ 925.0  850.0
+ 999.0  297.0
+ 608.0  846.0
+ 642.0  533.0
+ 656.0  646.0
+```
+
+### Metadata
+
+You may obtain meta data for a SAS data file using the `metadata` function.
+
+```julia
+julia> md = metadata("productsales.sas7bdat")
+SASLib.Metadata("productsales.sas7bdat", "US-ASCII", :LittleEndian, :none, 8192, 18, 1440, 10, Pair{Symbol,DataType}[:ACTUAL=>Float64, :PREDICT=>Float64, :COUNTRY=>String, :REGION=>String, :DIVISION=>String, :PRODTYPE=>String, :PRODUCT=>String, :QUARTER=>Float64, :YEAR=>Float64, :MONTH=>Float64])
+```
+
+It's OK to access the fields directly.
+```julia
+julia> fieldnames(md)
```

Comment:
I think to be compatible with 0.7 and onward, this line should read `fieldnames(typeof(md))`. There was a discussion on Slack or Discourse, can't remember which.